README.md

```diff
@@ -1 +1,172 @@
-# appointment-app
\ No newline at end of file
+### Table of Contents
+
+- [📖 About the Project](#about)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+### About  <a name="about-project"></a>
+
+CycleCruise is an app to create an appointment to test drive a motorcyle.
+
+## Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML(erb), CSS</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Frontend</summary>
+  <ul>
+    <li>React</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Backend</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Add appointment for a motorbike
+- Delete appointment for a motorbike
+- Create and Delete motorbikes
+
+
+<p align="right">(<a href="#table-of-contents">back to top</a>)</p>
+
+
+## Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: any web-browser.
+
+### Setup
+
+This API is consumed by its back-end counterpart [back-end](https://github.com/c00p75/appointment-app-backend)
+
+- Check the instructions to connect to the database in the [back-end](https://github.com/c00p75/appointment-app-backend) repo
+
+Clone this [repo](https://github.com/c00p75/appointment-app-frontend.git) to your desired folder.
+
+```
+    git clone https://github.com/c00p75/appointment-app-frontend.git
+```
+
+### Install
+
+Run from a terminal
+
+```
+    cd appointment-app-frontend
+    npm install
+```
+
+### Usage
+
+**To run the project follow the following instruction:-**
+
+```
+    npm start
+```
+
+### Run tests
+
+To run tests
+
+```
+  npm test
+```
```

Comment:
- [ ] You are including instructions to run tests, but the only test found is the one that comes with react, which is failing:

![image](https://github.com/c00p75/appointment-app-frontend/assets/57421823/7dbb54e2-f691-4445-9fa8-55114e558116)

Kindly, remove that file and also remove the "Run tests" instructions on the readme if you are not including tests for the frontend app. 👍